### DISCUSSION TITLE
gvm-tools

### DISCUSSION THREAD

**Discussion by Dhanush-3735** [immauss/openvas#248] [General]
Can I run Gvm commands as non interactive in your image?

**Comment by immauss** [comment]
Yes. 
If you are using scripts, you'll need to have them use the "tls" option.

example:

gvm-script --gmp-password admin --gmp-username admin tls ./scantest.py

You can do this from command line with docker exec 
```
docker exec -it openvas gvm-script --gmp-password admin --gmp-username admin tls ./scantest.py
````

Or you could get a shell in the running container and run them interactively there.
```
docker exec -it openvas bash 
root@4968cd4affa8:/#  gvm-script --gmp-password admin --gmp-username admin tls ./scantest.py
```
Of course you will need to either mount a directory with your scripts at container start, or copy them into the running container with:
``` docker cp```





Thanks,
Scott

**Comment by Dhanush-3735** [comment]
Thank you for your response. It is running as per your advised approach.
